core/definitions/html.tssl

```diff
@@ -2192,6 +2192,7 @@
       <element name='a' type='mixed'>
         <memberOf group='M_INLINE'/>
         <contains group='M_NOLINK'/>
+        <contains group='M_BLOCK'/>
```

Comment:
please add UTs for these 3 changes please

Reply:
I need the comments to explain why are these changes here

Reply:
done